packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -164,3 +170,67 @@ String _createProjectIdentifier(String name) {
   name = name.length == 0 ? 'untitled' : name;
   return 'com.yourcompany.$name';
 }
+
+final Set<String> _packageDependencies = new Set<String>.from(<String>[
+  'analyzer',
+  'args',
+  'async',
+  'barback',
+  'cassowary',
+  'charcode',
+  'collection',
+  'convert',
+  'crypto',
+  'csslib',
+  'flutter',
+  'glob',
+  'html',
+  'http_multi_server',
+  'http_parser',
+  'intl',
+  'logging',
+  'matcher',
+  'material_design_icons',
+  'mime',
+  'mojo',
+  'mojo_apptest',
+  'mojo_sdk',
+  'mojo_services',
+  'mojom',
+  'newton',
+  'package_config',
+  'path',
+  'petitparser',
+  'plugin',
+  'pool',
+  'pub_semver',
+  'quiver',
+  'shelf',
+  'shelf_static',
+  'shelf_web_socket',
+  'sky_engine',
+  'sky_services',
+  'source_map_stack_trace',
+  'source_maps',
+  'source_span',
+  'stack_trace',
+  'stream_channel',
+  'string_scanner',
+  'test',
+  'typed_data',
+  'utf',
+  'vector_math',
+  'watcher',
+  'yaml',
+]);
```

Comment:
This list seems hard to maintain.  Maybe we should pare this down to names that folks are likely to hit?  The only case I've actually heard of people hitting is "test".